src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,11 +4,32 @@
 
 package frc.robot.subsystems;
 
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
 
 public class Shooter extends SubsystemBase {
   /** Creates a new Shooter. */
-  public Shooter() {}
+  private CANSparkMax left;
+  private CANSparkMax right;
+
+  public Shooter() {
+    left = new CANSparkMax(Constants.SHOOTER_LEFT_ID, MotorType.kBrushless);
+    right = new CANSparkMax(Constants.SHOOTER_RIGHT_ID, MotorType.kBrushless);
+    right.setInverted(true);
+  }
+
+  public void setSpeed(double speed) {
+    left.set(speed);
+    right.set(speed);
+  }
+//one starts shooter motor, second one that pushes game piece into the shooter, turns of a few seconds after both are shooted.
```

Comment:
Since a comment was written here, I would recommend writing the comments in a style similar to the comments in LogitechGamingPad.java i.e. parameter types and names, return type, and the method's job. It seems that this comment describes the shooting command itself, so instead, only write about the function you are currently annotating.